code/datums/outfits/outfit_admin.dm

```diff
@@ -1309,7 +1309,7 @@
 
 	backpack_contents = list(
 		/obj/item/storage/box/survival = 1,
-		/obj/item/bikehorn = 1,
+		/obj/item/bikehorn/bwoinkhorn = 1,
```

Comment:
I would leave the Honksquad with both versions of the bikehorn, extra firepower for them.